src/main/java/agh/ics/oop/Animal.java

```diff
@@ -0,0 +1,46 @@
+package agh.ics.oop;
+
+public class Animal {
+
+    private MapDirection orientation;
+    private Vector2d position;
+
+    Vector2d lowerLeft = new Vector2d(0, 0);
```

Comment:
warto takie rzeczy wyciągać, formalnie są to stałe więc dobrze oznaczyć je słowami `static final` i nazwę napisać DUŻYMI_LITERAMI (będę o tym jeszcze wspominał na zajęciach)